:lady_beetle: [BUG] Scheduled message not working

**Describe the bug**
Trying to send a scheduled message either results in the app crashing, or opening the recently contacted list

**To Reproduce**
Steps to reproduce the behavior:
1. type something out
2. Click on + then schedule message
3. try to send
4. app crashes

**Expected behavior**
i would expect the app to schedule the message

**Smartphone (please complete the following information):**
 - Device: fairphone 4
 - OS: lineageos22 
 - App Version 4.1.1

**Screenshots**
If applicable, add screenshots to help explain your problem.

**Additional context**
Add any other context about the problem here.

FATAL EXCEPTION: RxCachedThreadScheduler-11
Process: dev.octoshrimpy.quik, PID: 10149
io.reactivex.exceptions.OnErrorNotImplementedException: Can't toast on a thread that has not called Looper.prepare()
Caused by: java.lang.NullPointerException: Can't toast on a thread that has not called Looper.prepare()
	at dev.octoshrimpy.quik.feature.compose.ComposeViewModel.bindView(SourceFile:0)
	at [[ ↑↑ Inferred subscribe point ↑↑ ]].(:0)
	at [[ ↓↓ Original trace ↓↓ ]].(:0)
	at com.android.internal.util.Preconditions.checkNotNull(Preconditions.java:174)
	at android.widget.Toast.getLooper(Toast.java:188)
	at android.widget.Toast.<init>(Toast.java:173)
	at android.widget.Toast.makeText(Toast.java:518)
	at android.widget.Toast.makeText(Toast.java:507)
	at android.widget.Toast.makeText(Toast.java:565)
	at dev.octoshrimpy.quik.common.util.extensions.ContextExtensionsKt.makeToast(SourceFile:0)
	at dev.octoshrimpy.quik.common.util.extensions.ContextExtensionsKt.makeToast$default(SourceFile:0)
	at dev.octoshrimpy.quik.feature.compose.ComposeViewModel$bindView$$inlined$withLatestFrom$18.apply(Unknown Source:357)
	at io.reactivex.internal.functions.Functions$Array4Func.apply(SourceFile:0)
	at io.reactivex.internal.functions.Functions$Array4Func.apply(SourceFile:0)
	at io.reactivex.internal.operators.observable.ObservableWithLatestFromMany$WithLatestFromObserver.onNext(Unknown Source:34)
	at io.reactivex.observers.SerializedObserver.onNext(Unknown Source:63)
	at io.reactivex.internal.operators.observable.ObservableWithLatestFrom$WithLatestFromObserver.onNext(Unknown Source:20)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.drainNormal(Unknown Source:47)
	at io.reactivex.internal.operators.observable.ObservableObserveOn$ObserveOnObserver.run(Unknown Source:8)
	at io.reactivex.internal.schedulers.ScheduledRunnable.run(Unknown Source:13)
	at io.reactivex.internal.schedulers.ScheduledRunnable.call(Unknown Source:0)
	at java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.util.concurrent.ScheduledThreadPoolExecutor$ScheduledFutureTask.run(ScheduledThreadPoolExecutor.java:347)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1145)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:644)
	at java.lang.Thread.run(Thread.java:1117)         



AHA here we go, i got logfox, and this popped up when i tried to send a scheduled message

--- a/presentation/src/main/java/com/moez/QKSMS/feature/compose/ComposeViewModel.kt
+++ b/presentation/src/main/java/com/moez/QKSMS/feature/compose/ComposeViewModel.kt
@@ -135,6 +135,7 @@ class ComposeViewModel @Inject constructor(
     private val searchSelection: Subject<Long> = BehaviorSubject.createDefault(-1)
 
     private var shouldShowContacts = threadId == 0L && addresses.isEmpty()
+    private var showScheduledToast = false
 
     private var bluetoothMicManager: BluetoothMicManager? = null
 
@@ -1118,7 +1119,7 @@ class ComposeViewModel @Inject constructor(
                         )
                     ).also {
                         newState { copy(scheduled = 0) }
-                        context.makeToast(R.string.compose_scheduled_toast)
+                        showScheduledToast = true
                     }
 
                     // sending a group message
@@ -1157,7 +1158,13 @@ class ComposeViewModel @Inject constructor(
                 )
             }
             .observeOn(AndroidSchedulers.mainThread())
-            .doOnNext { view.focusMessage() }
+            .doOnNext {
+                view.focusMessage()
+                if (showScheduledToast) {
+                    context.makeToast(R.string.compose_scheduled_toast)
+                    showScheduledToast = false
+                }
+            }
             .autoDisposable(view.scope())
             .subscribe()
 

